repair/repair.cc

```diff
@@ -1050,7 +1050,12 @@ static future<> repair_range(repair_info& ri, const dht::token_range& range) {
       }
       return ::service::get_local_migration_manager().sync_schema(ri.db.local(), neighbors).then([&neighbors, &ri, range, id] {
         return do_for_each(ri.table_ids.begin(), ri.table_ids.end(), [&ri, &neighbors, range] (utils::UUID table_id) {
-            auto cf = ri.db.local().find_column_family(table_id).schema()->cf_name();
+            sstring cf;
+            try {
+                cf = ri.db.local().find_column_family(table_id).schema()->cf_name();
```

Comment:
`cf` is not used, why keep it?

Reply:
It is used in logging. A table name is more readable than a uuid. It avoids looking up it from a table_id again and again.

Reply:
Sorry, I thought the patch introduced cf and did not use it, but it existed before.

Reply:
Ah, I see ;-)